src/Sylius/Bundle/CoreBundle/DependencyInjection/Compiler/CheckStatisticsOrdersTotalsProviderTypePass.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class CheckStatisticsOrdersTotalsProviderTypePass implements CompilerPassInterface
+{
+
+    public function process(ContainerBuilder $container): void
+    {
+        if (!$container->hasParameter('sylius_core.orders_statistics.intervals_map')) {
```

Comment:
Something that completely slipped my mind, but we should add some validation in the `Configuration` class as well.

`cannotBeEmpty` for both `interval` and `period_format` nodes, and custom validation for the interval which will check that it's a duration string.